custom_components/openhasp/__init__.py

```diff
@@ -601,10 +603,15 @@ async def async_push_image(
 
         cmd_topic = f"{self._topic}/command/{obj}.src"
 
-        rgb_image_url = (
-            f"{get_url(self.hass, allow_external=False)}/api/openhasp/serve/{image_id}"
-        )
-
+        if http_proxy == None:
```

Comment:
instead of testing against None change the if order

Reply:
Python is not my strongsuit, I understand your request but I am not sure (but suspect) if I can remove the '== None' and rely on python to understand that anything but 'None' is going to be okay to use... (with the actions reversed, obviously)

Also, it is a syntax thing, while it is perfectly functional... but if you can confirm my suspicion above, I will fix it.

Reply:
ok, change applied :)